resources/web/style/guide_base_styles.pcss

```diff
@@ -0,0 +1,25 @@
+#guide {
+  * {
+    font-family: Inter, sans-serif;
+  }
+
+  hr {
+    margin: 1em 0;
+  }
+
+  dt span {
+    font-weight: bold;
+
+    code {
+      font-size: 16px;
+    }
+  }
+
+  dd {
+    margin: 0 0 0.5em 2em;
+  }
+
+  ul.lists li {
```

Comment:
I think we have `list.pcss` for this one. It'd probably be worth tracking down if we actually use it though.

Reply:
I scanned and couldn't find any references, so it's dropped now.